index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>SPA - Pet Food</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <table class="food-table" rules="groups">
```

Comment:
where did you find rules? Its been deprecated 👎 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table#attr-rules